src/Schema/OwnerType.ts

```diff
@@ -62,5 +62,8 @@ export type PageOwnerType =
   | OwnerType.collection
   | OwnerType.consign
   | OwnerType.fair
+  | OwnerType.gene
```

Comment:
@abhitip - wasn't sure what contexts this will be used in, but we may need to expand available `PageOwnerTypes` for web while implementing.

Reply:
Yes we can expand as we go!